### PR TITLE
Sec 2.4 Make stdout redirection easier to spot

### LINKED ISSUE #79: Section 2.4.2: Make the stdout redirection a little more obvious in this section
`sections/verdi_cmdline.html#structuredata`

In this section, I feel like the flow of the tutorial is to delete what's in `<>` it happened to me twice that I remove the `>` stdout redirection operator, I read this

```bash
$ verdi data structure export --format xsf <IDENTIFIER> > <IDENTIFIER>.xsf
```

we might want to improve here providing an actual uuid, maybe we could include a comment and a command like:

```bash
# verdi data structure export --format xsf <IDENTIFIER> > <IDENTIFIER>.xsf
verdi data structure export --format xsf 254e5a86 > 254e5a86.xsf
```

The uuids would only need to be replaced if we ever change our export.

### PR DESCRIPTION
**Pull request by odarbelaeze**
Fix #79

### CODE REVIEW

File: docs/pages/2019_MARVEL_Psik_MaX/sections/verdi_cmdline.rst [hunk 1]
```diff
@@ -365,9 +365,10 @@ You should be able to rotate the view with the right mouse button.
 Alternatively, especially if showing them interactively is too slow over SSH, you can export the content of a structure node in various popular formats such as ``xyz`` or ``xsf``.
 This is achieved by typing in the terminal:
 
-.. code:: console
+.. code:: bash
```
Comment (ltalirz) [inline docs/pages/2019_MARVEL_Psik_MaX/sections/verdi_cmdline.rst:407]: this should remain "console" (used for uniform styling of the background)

**Review by ltalirz — CHANGES_REQUESTED** [review]
thanks, please just revert the `code` type

File: docs/pages/2019_MARVEL_Psik_MaX/sections/verdi_cmdline.rst (hunk 1 above)
Comment (odarbelaeze) [reply docs/pages/2019_MARVEL_Psik_MaX/sections/verdi_cmdline.rst:407]: You might be aware, but this is how it looks with bash,

console

It's pretty clear that the first line is a comment and the second one you should use, conversely this is how it looks with bash:

actual console

To my eyes the comment is hardly distinguishable, so I'm okay with 3 different options,

1. Sacrificing consistency for the sake of clarity and not make an update.
2. Sacrificing clarity for the sake of consistency and update it to `console`.
3. Getting consistency and clarity through work, and update all the consoles to bash, and make sure python code blocks look different enough.

After stating what's obvious to me, I'd appreciate you help me pick between these three equally valuable options.

Comment (ltalirz) [reply docs/pages/2019_MARVEL_Psik_MaX/sections/verdi_cmdline.rst:407]: oh, sorry - I wasn't aware of this difference, thanks for pointing it out!
I think 3. is the right approach - I'll take care of this (but merge your PR first).

**Pull request merged**